sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs

```diff
@@ -3138,14 +3105,47 @@ public partial class VnetInfoResourceData : Azure.ResourceManager.Models.Resourc
     {
         public VnetInfoResourceData() { }
         public string CertBlob { get { throw null; } set { } }
-        public string CertThumbprint { get { throw null; } }
+        public System.BinaryData CertThumbprint { get { throw null; } }
         public string DnsServers { get { throw null; } set { } }
         public bool? IsResyncRequired { get { throw null; } }
         public bool? IsSwift { get { throw null; } set { } }
         public string Kind { get { throw null; } set { } }
-        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.AppService.Models.VnetRoute> Routes { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.AppService.Models.AppServiceVnetRoute> Routes { get { throw null; } }
         public Azure.Core.ResourceIdentifier VnetResourceId { get { throw null; } set { } }
     }
+    public partial class WebAppBackupData : Azure.ResourceManager.Models.ResourceData
+    {
+        public WebAppBackupData() { }
+        public int? BackupId { get { throw null; } }
+        public string BlobName { get { throw null; } }
+        public string CorrelationId { get { throw null; } }
```

Comment:
I see `CorrelationId` in another place is `Guid`. This should probably also be `Guid`?